test/alternator/test_batch.py

```diff
@@ -364,10 +364,12 @@ def test_batch_get_item_large(test_table_sn):
     # UnprocessedKeys), so for reliable success of this test we need a loop:
     responses = []
     to_read = { test_table_sn.name: {'Keys': [{'p': p, 'c': c} for c in range(count)], 'ConsistentRead': True } }
+    maybe_inject_error(dynamodb, "alternator_batch_get_item", one_shot=True)
```

Comment:
As I noted in the previous patch, instead of a "maybe" you can duplicate this test (either really duplicate it or a part of it, or use `@pytest.mark.parametrize` - we haven't used it in alternator tests yet, but the dtest guys have been using it heavily) - and have the test with inject_error() be automatically skipped when it can't be done.

Reply:
I could, but note that it's still a "maybe", because it will depend on how Scylla was compiled. Although in this case it's easy to test that - if error injection is disabled, the list of registered injection points will still be empty after registering such a point via API, and then the test case can be skipped. So maybe I'll indeed duplicate this test after all.